internal/datastore/crdb/pool/health.go

```diff
@@ -44,7 +46,10 @@ func NewNodeHealthChecker(url string) (*NodeHealthTracker, error) {
 	return &NodeHealthTracker{
 		connConfig:    connConfig,
 		healthyNodes:  make(map[uint32]struct{}, 0),
-		nodesEverSeen: make(map[uint32]struct{}, 0),
+		nodesEverSeen: make(map[uint32]*rate.Limiter, 0),
+		newLimiter: func() *rate.Limiter {
+			return rate.NewLimiter(rate.Every(1*time.Minute), 2)
```

Comment:
can't check that `2` means without looking into `NewLimiter` docs, mind documenting it with a constant?